website/content/en/getting-started/usage/_index.md

```diff
@@ -51,6 +51,7 @@ To provide identity to access key vault, refer to the following [section](#provi
             objectAlias: SECRET_1           # [OPTIONAL available for version > 0.0.4] object alias
             objectType: secret              # object types: secret, key or cert. For Key Vault certificates, refer to https://azure.github.io/secrets-store-csi-driver-provider-azure/configurations/getting-certs-and-keys/ for the object type to use
             objectVersion: ""               # [OPTIONAL] object versions, default to latest if empty
+            permission: 0755                # [OPTIONAL] permission for secret file being mounted into the pod, default is 0644 if not specified.
```

Comment:
Isn't this supposed to be [filePermission](https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/751/files#diff-334716bb990e31490cd8bb9bee9c34757d228c884e35663d88bd26889ffa105bR102)?